packages/create-astro/src/index.ts

```diff
@@ -181,7 +161,9 @@ export async function main() {
 		await Promise.all([
 			...FILES_TO_REMOVE.map(async (file) => {
 				const fileLoc = path.resolve(path.join(cwd, file));
-				return fs.promises.rm(fileLoc);
+				if (fs.existsSync(fileLoc)) {
+					return fs.promises.rm(fileLoc, {});
+				}
 			}),
 			...POSTPROCESS_FILES.map(async (file) => {
```

Comment:
Is this loop even necessary now? Couldn't we just remove all the `CHANGELOG.md` inside the examples?

Reply:
You know... I think you're right. Removing!